data-processing/tests/mathml/x_sub_four_sub_three.xml

```diff
@@ -0,0 +1,17 @@
+<span class="katex">
+    <math
+        xmlns="http://www.w3.org/1998/Math/MathML">
+        <semantics>
+            <mrow>
+                <msub s2:index="5">
+                    <mi s2:start="0" s2:end="1" s2:index="0">x</mi>
+                    <msub s2:index="4" s2:start="2" s2:end="7">
+                        <mn s2:start="3" s2:end="4" s2:index="1">4</mn>
+                        <mn s2:start="5" s2:end="6" s2:index="2">3</mn>
+                    </msub>
+                </msub>
+            </mrow>
+            <annotation encoding="application/x-tex">x_{t_i}</annotation>
```

Comment:
nit pick: can you update this element to have the value `x_43`? It's not important for the sake of the test running, though might avoid future confusion about this test data.